services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/mapper/BeanTableSchema.java

```diff
@@ -265,12 +281,14 @@ private BeanTableSchema(StaticTableSchema<T> staticTableSchema) {
                 return EnhancedType.documentOf(
                     (Class<Object>) clazz,
                     (TableSchema<Object>) ImmutableTableSchema.recursiveCreate(clazz, metaTableSchemaCache),
-                    b -> b.preserveEmptyObject(preserveEmptyObject));
+                    b -> b.preserveEmptyObject(attributeConfiguration.preserveEmptyObject())
+                          .ignoreNulls(attributeConfiguration.ignoreNulls()));
```

Comment:
This makes me wonder if we should find a way to just have them share the configuration object. Imagine when we have 20 of these...

Reply:
Yeah, created a variable for this